knowledge-base/the-remote-server-returned-an-error-401-unauthorized-when-using-a-pictureBox-from-url.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Picturebox Throws Error 401 Unauthorized
+description: The Error 401 Unauthorized might be thrown if you are using a PictureBox with a Value from URL
+type: troubleshooting
+page_title: The remote server returned an error:(401) Unauthorized when using a PictureBox from URL
+slug: the-remote-server-returned-an-error-401-unauthorized-when-using-a-pictureBox-from-url
+position: 
+tags: picturebox,error401
+ticketid: 1412798
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+    <tr>
+			<td>Report item</td>
+			<td>PictureBox</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The error might be thrown when the report contains a [PictureBox](./report-items-picture-box) whose Value is a URL. 
+In general, when you provide an URL as a PictureBox.Value, we use a WebClient to download the image.
+At that point, the connection to server fails with the statement that the application's process is not authorized to get the resource.
+
+## Error Message
+The remote server returned an error:(401) Unauthorized.
+
+## Solution
+
+You can to create a [User Function](.expressions-user-functions) that will download the image after authorizing, and returns it (e.g. as type [Bitmap](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.bitmap?view=netframework-4.8)) for the picture box value.
```

Comment:
It may be more efficient to return it as a byte[]. You may add both possibilities.